docs/advanced_topics/settings.rst

```diff
@@ -250,15 +250,20 @@ Images
 This setting lets you provide your own image model for use in Wagtail, which might extend the built-in ``AbstractImage`` class or replace it entirely.
 
 
-Maximum Upload size for Images
-------------------------------
-
 .. code-block:: python
 
     WAGTAILIMAGES_MAX_UPLOAD_SIZE = 20 * 1024 * 1024  # i.e. 20MB
 
 This setting lets you override the maximum upload size for images (in bytes). If omitted, Wagtail will fall back to using its 10MB default value.
 
+.. code-block:: python
+
+    WAGTAILIMAGES_FEATURE_DETECTION_ENABLED = True
+
+This setting enables feature detection once OpenCV is installed, see all details on the :ref:`image_feature_detection` documentation.
+
+The ``WAGTAILIMAGES_BACKENDS`` setting is no longer supported, it was introduced in 0.3.
```

Comment:
I don't think there's much point in mentioning `WAGTAILIMAGES_BACKENDS` - it was dropped in Wagtail 1.0, which is ancient history now :-)